src/mixins/input.js

```diff
@@ -68,8 +68,13 @@ export default {
   mounted () {
     this.$nextTick(() => {
       const input = this.$refs.input
-      if (this.autofocus && input) {
-        input.focus()
+      if (input) {
```

Comment:
I think the whole block should look like this:
```
if (this.autofocus && input) {
  input.focus()
  if (this.autofocus === 'select') {
    input.select()
  }
}
```

Reply:
Done